content/posts/2021-05-airdrop-anywhere-part-2.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "AirDrop Anywhere - Part 2 - Writing some code"
+date: 2021-05-04T15:00:00Z
+tags: [.net, networking, airdrop, apple]
+images: [img/airdrop-anywhere-cover.jpg]
+---
+
+> This is part 2 of a series of posts:
+> - [Part 1 - Introduction](/posts/2021-05-airdrop-anywhere-part-1)
+> - [Part 2 - Writing some code](/posts/2021-05-airdrop-anywhere-part-2)
+> - [GitHub (mDNS bits)](https://github.com/deanward81/AirDropAnywhere/tree/2021-05-04-mdns)
+> - [GitHub (latest)](https://github.com/deanward81/AirDropAnywhere/tree/main) - **NOTE** still work in progress!
```

Comment:
Note: TOC has diverged here (2 additions) - intentional? If not, any way to share the include partial or something?

Reply:
Heh, I can't work out how to do a partial in Hugo - divergence in this case is intentional - wanna have people read the intro before digging into the codez :)

Reply:
Suggestion: make the last 2 items nested under the post 1 level in, looks intentional and declares them as sub-resources clearly